qt/scientific_interfaces/Inelastic/Analysis/FunctionBrowser/SingleFunctionTemplateBrowser.cpp

```diff
@@ -56,9 +57,9 @@ void SingleFunctionTemplateBrowser::createProperties() {
   m_presenter.init();
 }
 
-void SingleFunctionTemplateBrowser::setDataType(const QStringList &allowedFunctionsList) {
+void SingleFunctionTemplateBrowser::setDataType(std::vector<std::string> const &allowedFunctionsList) {
   ScopedFalse _false(m_emitEnumChange);
-  m_enumManager->setEnumNames(m_fitType, allowedFunctionsList);
+  m_enumManager->setEnumNames(m_fitType, convertVectorToQStringList(allowedFunctionsList));
```

Comment:
In these 2 functions, is it necessary to add the prefix convert to the function name?
VectorToQStringList, or QStringListToVector is not ambiguous, right?